logzio/resource_alert_v2.go

```diff
@@ -292,9 +294,19 @@ func resourceAlertV2Create(d *schema.ResourceData, m interface{}) error {
 	alertId := strconv.FormatInt(a.AlertId, BASE_10)
 	d.SetId(alertId)
 
-	// A temporary solution to handle a 404 error that happens otherwise
-	time.Sleep(delayGet * time.Second)
-	return resourceAlertV2Read(d, m)
+	return retry.Do(
+		func() error {
+			return resourceAlertV2Read(d, m)
+		},
+		retry.RetryIf(
+			func(err error) bool {
+				if strings.Contains(err.Error(), "missing alert"){
+					return true
+				}
+				return false
+			}),
+		retry.Delay(delayGetAlertV2),
```

Comment:
If you want, you could do delayFunc so the delays won't be a second each but 1,2,4.. or 1,1,2,3,5.. 
Anyway, please set max attempts for the retries